src/_cffi_src/openssl/ssl.py

```diff
@@ -677,6 +691,11 @@
 
 #if CRYPTOGRAPHY_OPENSSL_LESS_THAN_111
 static const long Cryptography_HAS_TLSv1_3 = 0;
+/*
+don't set this to 0,
+SSL_CTX_set_min_proto_version(TLS1_3_VERSION) would silently allow all version
+*/
```

Comment:
This doesn't matter, since we make the constant inaccessible when it's not available. Please make this consistent with all the other constants and remove the comment.